index.js

```diff
@@ -140,6 +140,26 @@ Atrium.endpoints = [
     url: '/users/:userGuid/accounts/:accountGuid/transactions',
     clientMethod: 'listAccountTransactions'
   },
+  {
+    method: 'post',
+    url: '/users/:userGuid/members/:memberGuid/verify',
+    clientMethod: 'verifyMember'
+  },
+  {
+    method: 'get',
+    url: '/users/:userGuid/members/:accountOrMemberGuid/account_numbers',
```

Comment:
This endpoint is only for getting all of the account numbers for a _member_; it's not overloaded. There is a separate endpoint for getting account numbers for an _account_:

>/users/:userGuid/accounts/:accountGuid/account_numbers